.github/workflows/release.yml

```diff
@@ -167,16 +164,13 @@ jobs:
     steps:
       - name: Make checkout speedy
         run: git config --global fetch.parallel 50
-      - name: Checkout
```

Comment:
@aryx my only comment here is to ask why we're removing the names of the steps? I find them to be useful markers especially in longer workflows.

Reply:
It's just repeating the uses:/run: that is just after.
It's like writing code like: ```
// Calling foo()
foo()
```
For steps where the run: contains a long series of commands, then I keep the name: because it's actually a useful comment that gives an overiew of what this step is doing, but otherwise it's just noise in the file.